src/browser/browser.js

```diff
@@ -4,7 +4,7 @@
   'use strict';
 
   const {html, node, render, cache} = require('reflex');
-  const {Record, Any, Union, Maybe} = require('../common/typed');
+  const {Record, Maybe} = require('typed-immutable');
```

Comment:
I have submitted bug for this #625 and will factor out those changes into separate pull.